stf/src/sgx.rs

```diff
@@ -138,21 +203,23 @@ impl Stf {
                 Ok(())
             }
             TrustedCall::balance_unshield(account_incognito, beneficiary, value, shard) => {
+                //let call_hash = blake2_256(&call.encode());
```

Comment:
Can we delete this?

Reply:
Will do some clean up, thanks for pointing it out.

Reply:
Deleted